### ISSUE TITLE
Color Banding

### ISSUE THREAD

**Issue by jochenschmich** [moonlight-stream/moonlight-qt#497]
I'm having a problem with color banding. Posted in another issue - but I thought this might be caused by something different or should be handled separately.

Moonlight really works perfect, except for the color banding.
Using GTX 1080 / Driver 456.71 on the host side.
I usually see major banding in games in a sky at day and especially at night.

This is how it looks like on the client with Moonlight:
![image](https://user-images.githubusercontent.com/76489910/103572363-ba404080-4ecc-11eb-86c9-5fe0042d89c3.png)

This is how it looks like when I simply take a screenshot without Moonlight:
![image](https://user-images.githubusercontent.com/12835994/104816996-d7ef9d00-581e-11eb-997b-74f5a6501ae9.png)

_Originally posted by @sneakpeekdev in https://github.com/moonlight-stream/moonlight-qt/issues/317#issuecomment-754174306_

**Comment by cgutman** [comment]
I think the banding is due to 8-bit color depth as detailed by Intel here: https://software.intel.com/content/www/us/en/develop/articles/enable-10bpp.html

It’s possible to get 10-bit HEVC from GeForce Experience when operating in HDR mode, but I don’t think it’s possible to request HEVC Main 10 for non-HDR games. But I can try it and see what happens.

**Comment by jochenschmich** [comment]
Hi there, thanks, this would be quite interesting!
With the stream it looks a bit worse, than if simply rendered on a 8 bit display.

**Comment by witchcraftsman** [comment]
Up with some news:
https://www.theverge.com/2021/3/17/22336240/microsoft-windows-10-auto-hdr-pc-gaming-directx-11-12
Will that affect Moonlight HDR experience as well?

**Comment by Dantido2** [comment]
This is most likely an issue with the fact that GeForce Experience uses YUV424 on their streams. This makes the color palette ultra limited compared to native.
The only way to get a better color palette that resembles the native display, unfortunately, is to pay for Parsec Premium, and well, you know how Parsec is...

**Comment by ThreePinkApples** [comment]
Not sure if it is the same issue, but I've discovered that when running the Moonlight client on an Nvidia SHIELD , where I've turned on Dolby Vision, the output is stuck at 12bit 4:2:2 when playing HDR games, which causes colour banding in AC Valhalla for me. If I turn off Dolby Vision in the SHIELD settings menu, the output is 10bit 4:2:0 and the banding is gone. Other apps (such as Plex) correctly change the output to 10bit when playing HDR10 movies, so this seems like something Moonlight also could, and should, do.
